src/Constants.h

```diff
@@ -133,26 +134,46 @@ constexpr int H264_RF_CONSTANT = 40;
 // Arm inverse kinematics
 namespace arm {
 
+constexpr double MAX_EE_VEL = 0.3; // m/s
```

Comment:
Instead of putting `// m/s` at the end, can we add proper doxy comments saying "Maximum commanded end-effector velocity, in m/s" or something